packages/perseus/src/validation.types.ts

```diff
@@ -33,6 +33,12 @@ import type {Relationship} from "./widgets/number-line/number-line";
 
 export type UserInputStatus = "correct" | "incorrect" | "incomplete";
 
+// TODO (LEMS-2396): remove validation logic from widgets that don't validate
+export type EmptyUserInput = Empty;
+
+// TODO (LEMS-2396): remove validation logic from widgets that don't validate
+export type NullUserInput = null;
```

Comment:
Decided to consolidate these since they're the same and will hopefully be going away in an upcoming PR.